joint_limits:
  joint1:
    has_position_limits: false
    min_position: 150.0
    max_position: 228.0
    has_velocity_limits: true
    max_velocity: 20. #56.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: false
    max_effort: 1.0